packages/core/strapi/lib/services/content-api/index.js

```diff
@@ -34,18 +50,11 @@ const createContentAPI = (/* strapi */) => {
     Promise.all(
       uniq(actions).map((action) =>
         providers.action.register(action).catch(() => {
-          // console.log('Key already exists', action);
+          console.warn('Trying to add action that already exists', action);
```

Comment:
Maybe we could use `strapi.log.warn` here to benefit from the user's Winston configuration, wdyt?

Reply:
Yes, I didn't know that existed, thanks 😆